.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_Store
 .idea
 .phpunit.*
+.php_cs.cache
```

Comment:
I'm confused here - why do we need to specifically ignore something that I didn't think we should be creating any more?

Reply:
Well, yep we should only need this or the line below, but I figured less harm to have both until our PHP CS Fixer version settles.